project_report/tex/theory_and_methods.tex

```diff
@@ -1,5 +1,53 @@
 \section{Theory \& Methods}
 
+\subsection{Theory}
+
+\subsubsection{Error Function}
+The Error Function describes the probability that an observation lies within
+a given interval, and can be seen in equation \ref{eq:1}.
+\begin{equation} \label{eq:1}
+    Erf(x)=\frac{2}{\sqrt{\pi}} \int_{-\infty}^x e^{-t^2} dt
+\end{equation}
+
+Where \(t=\frac{x}{\sqrt{\pi}}\). The error function is found by integrating
+the normalised Gaussian distribution. The function is normalised by the
+coeffecient in front of the integral, which means that \(\frac{2}{\sqrt{\pi}}\)
```

Comment:
Is this sentence complete? if so, it ends kinda weird. "which means that \(\frac{2}{\sqrt{\pi}}\) --IS SOMETHING MISSING HERE?-- A visualisation..."